lms/static/js/search/views/search_item_view.js

```diff
@@ -18,7 +18,12 @@ define([
         },
 
         initialize: function () {
-            this.tpl = _.template($('#search_item-tpl').html());
+            if(this.model.get('content_type') === "sequence") {
```

Comment:
This is fine, but I think I would have been more inclined to use a variable for the template name - something like this:
```
var template_name = (this.model.get('content_type') === "sequence") ? '#search_item_seq-tpl' : '#search_item-tpl';
this.tpl = _.template($(template_name).html());
```